### PR TITLE
fixed style3 inactive color

### PR DESCRIPTION
**Pull request by xxJeevesxx**
style 3 was not correctly applying inactive color

### CODE REVIEW

File: lib/nav-bar-styles/style-3-bottom-nav-bar.widget.dart [hunk 1]
```diff
@@ -60,11 +60,17 @@ class BottomNavStyle3 extends StatelessWidget {
...
-                                    color: (item.activeColorSecondary == null
-                                        ? item.activeColorPrimary
-                                        : item.activeColorSecondary),
+                                    color: isSelected
+                                        ? (item.activeColorSecondary == null
+                                            ? item.activeColorPrimary
+                                            : item.activeColorSecondary)
+                                        : item.inactiveColorPrimary == null
+                                            ? item.activeColorPrimary
+                                            : item.inactiveColorPrimary,
```
Comment (jb3rndt) [inline lib/nav-bar-styles/style-3-bottom-nav-bar.widget.dart:73]:
```suggestion
                                        : item.inactiveColorSecondary == null
                                            ? item.inactiveColorPrimary
                                            : item.inactiveColorSecondary,
```

File: lib/nav-bar-styles/style-3-bottom-nav-bar.widget.dart [hunk 2]
```diff
@@ -60,11 +60,17 @@ class BottomNavStyle3 extends StatelessWidget {
                                         ? (item.activeColorSecondary == null
                                             ? item.activeColorPrimary
                                             : item.activeColorSecondary)
-                                        : item.inactiveColorPrimary)
+                                        : item.inactiveColorPrimary == null
+                                            ? item.activeColorPrimary
+                                            : item.inactiveColorPrimary)
```
Comment (jb3rndt) [inline lib/nav-bar-styles/style-3-bottom-nav-bar.widget.dart:65]:
```suggestion
                                        : item.inactiveColorSecondary == null
                                            ? item.inactiveColorPrimary
                                            : item.inactiveColorSecondary )
```

**Review by jb3rndt — COMMENTED** [review]
This will probably be a problem in other styles too but it will be reworked in #42 anyway... The only thing is that we should conform to the documentation which uses the secondary color for the text and defaults to the primary color:

**Comment by xxJeevesxx** [comment]
Oops, good catch on the edits.

I didn't see the other PR. Should I check out the other styles too or not worry about it since it's getting updating in #42?

**Comment by jb3rndt** [comment]
I will fix them in #42, thanks for contributing :)

**Pull request merged**